instance/tests/api/test_instance.py

```diff
@@ -82,22 +85,20 @@ def test_get_domain(self):
         self.assertIn(('url', 'http://domain.api.example.com/'), instance_data)
         self.assertIn(('studio_url', 'http://studio.domain.api.example.com/'), instance_data)
 
-    def test_provision_waiting_for_server(self):
+    def test_provision_not_allowed(self):
         """
-        POST /:id/provision - Instance is waiting for server
+        POST /:id/provision - Instance is waiting for server or configuring server
         """
+        instance_factories = (
+            WaitingSingleVMOpenEdXInstanceFactory, ConfiguringSingleVMOpenEdXInstanceFactory
```

Comment:
Nit: better to use `ddt`. Though you'd need to add it. I'm surprised, but despite the fact that OpenCraft IM has 110 requirements listed in requirements.txt it is not currently using `ddt`.